app/views/users/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<h2>Create a New User</h2>
+
+<%= form_tag "/users", method: "create" do %>
+
+<%= label_tag :first_name %>
+<%= text_field_tag :first_name %>
+
+<%= label_tag :last_name %>
+<%= text_field_tag :last_name %>
+
+<%= label_tag :email %>
+<%= text_field_tag :email %>
+
+<%= label_tag :password %>
```

Comment:
Is there a password tag we can use to hide the password when it is being entered?

Reply:
`password_field_tag` hides the password (makes it look like `........`)

Reply:
added in a password field tag to hide password